playground/react/index.tsx

```diff
@@ -8,15 +8,17 @@ import Page from './components/page';
 import Feature from './components/feature';
 
 import { apiPlugin, storyblokInit } from '@storyblok/react';
+import IFrameEmbed from './components/iframe-embed';
 
 storyblokInit({
-  accessToken: 'd6IKUtAUDiKyAhpJtrLFcwtt',
+  accessToken: 'OurklwV5XsDJTIE1NJaD2wtt',
```

Comment:
also changing the demo space here, are we sure that is correct or should we keep the original demo space